modulesets/gstreamer.modules

```diff
@@ -97,6 +97,15 @@
     <branch repo="slomo" module="gst-player" checkoutdir="gst-player"/>
   </autotools>
 
+  <autotools id="gst-editing-services" autogenargs="--disable-gtk-doc">
+    <dependencies>
+      <dep package="libxml2"/>
+      <dep package="gstreamer"/>
+      <dep package="gst-devtools"/>
```

Comment:
gst-editing-services does not have a dependency with gst-validate (what in gst-devtools), but it is useful if build with gst-validate, so I moved it to suggests from dependency.
In addition, it has a dependency with gst-plugins-base, so I add it also.